### PR TITLE
[stable-v2.2] topology1: Add topology for 4ch DMIC+ SSP codec+HDMI-in capture..

### PR DESCRIPTION
**Pull request by CBala21**
Add support to generate the topology which can be used to support 
4channel DMIC,SSP based codec and HDMI-in capture via I2S combo.

### CODE REVIEW

File: tools/topology/topology1/CMakeLists.txt [hunk 1]
```diff
@@ -316,6 +316,7 @@ set(TPLGS
```
Comment (kv2019i) [inline tools/topology/topology1/CMakeLists.txt:316]: I think the commit should describe the change a bit more. You can just say add support for new sof-adl-es8336-ssp1-hdmi-ssp02-4ch topology and explain in git commit where this is needed (4ch variant of the SSP codec and HDMI-in capture combo).

Comment (CBala21) [reply tools/topology/topology1/CMakeLists.txt:316]: Updated the commit message a suggested. Please check. Thanks.

**Comment by marc-hb** [comment]
Commit titles should pack as much information as possible in a very limited space. Example:

> Add cmake changes to support the generation of the topology which can be used to support X, Y Z...

=> Add topology for X, Y, Z...

(this is not really CMake code, it's just a list that happens to be implemented in CMake)

**Review by plbossart — APPROVED** [review]

**Comment by marc-hb** [comment]
Also, for review convenience, please update the Pull Request description and keep it in sync when force-pushing a new commit message.

Github is not really designed for force-pushes (no `git range-diff`)
https://docs.zephyrproject.org/latest/contribute/contributor_expectations.html#workflow-suggestions-that-help-reviewers

**Comment by CBala21** [comment]
> Commit titles should pack as much information as possible in a very limited space. Example:
> 
> > Add cmake changes to support the generation of the topology which can be used to support X, Y Z...
> 
> => Add topology for X, Y, Z...
> 
> (this is not really CMake code, it's just a list that happens to be implemented in CMake)

Updated the commit message and PR description as you suggested. please check.

**Comment by marc-hb** [comment]
@wszypelt , @lrudyX : very worrying build failure in https://sof-ci.01.org/sof-pr-viewer/#/build/PR8993/build13802926, can you please take a look?

```
west init -l sof
23:22:30,371 DEBUG - Command working directory: /buildagent/workspace/24733
23:22:30,518 WARN  - FATAL ERROR: can't init: no west.yml found in /buildagent/workspace/24733/sof
```

> Updated the commit message and PR description as you suggested. please check.

Looks good, thanks for the extra effort!

**Review by kv2019i — APPROVED** [review]

**Pull request merged**